src/DynamoCoreWpf/Views/PackageManager/InstalledPackagesView.xaml

```diff
@@ -165,32 +166,21 @@
                                             <Label Content="{x:Static p:Resources.InstalledPackageViewCustomNodesLabel}"
                                                    FontWeight="Bold"
                                                    Foreground="Gray"></Label>
-                                            <ListBox Name="LoadedCustomNodes"
+                                            <ItemsControl Name="LoadedCustomNodes"
                                                      ItemsSource="{Binding Path=Model.LoadedCustomNodes}"
                                                      BorderThickness="0"
                                                      Padding="0"
                                                      Margin="10,0,0,0"
                                                      Background="Transparent">
 
-                                                <ListBox.ItemContainerStyle>
```

Comment:
BTW, this `ItemContainerStyle` tries to overwrite default ListBox style and use transparent brushes. And it's never used selection of item(in `ListBox` it's possible to select items). So, I assume, that  here must be `ItemsControl` instead of `ListBox`.